#!/bin/sh -e
## 20-kolabquotawarn-issue851.dpatch by Steffen Joeris <steffen.joeris@skolelinux.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This is the patch from the rpm

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- kolabd-1.9.4/kolabquotawarn.orig	2005-07-23 00:50:20 +0200
+++ kolabd-1.9.4/kolabquotawarn	2005-09-14 14:31:18 +0200
@@ -34,6 +34,7 @@
 use IO::File;
 use DB_File;
 use Mail::Message;
+use Mail::Transport::SMTP;
 use Cyrus::IMAP::Admin;
 use Kolab;
 use Kolab::Util;
